http-serverless/samples/sample1/src/main/java/io/micronaut/samples/httpserverless/sample1/Application.java

```diff
@@ -0,0 +1,24 @@
+package io.micronaut.samples.httpserverless.sample1;
+
+import io.micronaut.runtime.Micronaut;
+
+/**
+ * This program demonstrates how to use Micronaut HTTP Router without Netty.
+ * It reads HTTP requests from stdin and writes HTTP responses to stdout.
+ *
+ * @author Sahoo.
+ */
+public class Application {
+
+    public static void main(String[] args) throws Exception {
+        // Need to disable banner because Micronaut prints banner to STDOUT,
+        // which gets mixed with HTTP response.
+        // See GCN-4489
```

Comment:
better to define a `@ContextConfigurer` for this.

Reply:
I am not familiar with Micronaut annotations. I am sure what you are suggesting is better than what I have drafted. We will fix it before merging.

Reply:
In the past I've just done System.setOut(System.err). That way we don't risk user code printing something to stdout, and we fix logback at the same time without a config change.

Reply:
STDOUT can't always be bound to STDERR because traditional UNIX compatible programs expect output in STDOUT. e.g,, Unix shell's pipe functionality or inetd, etc. This is especially true in this case.